beets/util/__init__.py

```diff
@@ -949,12 +949,21 @@ def interactive_open(targets, command):
     """
     assert command
 
-    # Split the command string into its arguments.
-    try:
-        args = shlex.split(command)
-    except ValueError:  # Malformed shell tokens.
+    if os.name == 'nt':
         args = [command]
 
+        if command == 'cmd.exe':
+            targets = ['/C start ' + targets[0]]
+
+    else:
+        # Split the command string into its arguments.
+        try:
+            # Doesn't work well with Windows
+            # (removes all the '\' from path resulting in an invalid path)
+            args = shlex.split(command)
+        except ValueError:  # Malformed shell tokens.
+            args = [command]
+
     args.insert(0, args[0])  # for argv[0]
```

Comment:
According to https://github.com/beetbox/beets/issues/2847#issuecomment-375387406, that should only be done on *nix, is this still the case?